server/routes/shop.js

```diff
@@ -283,4 +292,20 @@ router.get("/items", async function(req, res) {
   }
 });
 
+router.get("/getitem/:itemid", async (req, res, next) => {
```

Comment:
this is not a good RESTful naming convention - https://restfulapi.net/resource-naming/

Reply:
Thanks for pointing this out. I have made the changes and keep modifying the ones where I didn't follow this convention.